salt/modules/ini_manage.py

```diff
@@ -50,6 +50,10 @@ def set_option(file_name, sections=None):
         If the Ini does not contain sections the keys and values represent the
         options
 
+    separator : =
+        A character used to separate keys and values. Standard ini files use
+        the "=" character.
+
```

Comment:
@boltronics, the documentation on this option should also include a `versionadded`:
```rst
    separator : =
        A character used to separate keys and values. Standard ini files use
        the "=" character.

        .. versionadded:: Carbon
```